src/Illuminate/Console/Scheduling/ManagesFrequencies.php

```diff
@@ -274,6 +274,31 @@ public function monthlyOn($day = 1, $time = '0:0')
     }
 
     /**
+     * Schedule the event to run on the last day of the month at a given time.
+     *
+     * @param  string  $time
+     * @return $this
+     */
+    public function endOfMonth($time = '0:0')
```

Comment:
Indeed, it may be a method without a justifiable use case. I wasn't thinking so much reports for this one as notifications... end of the month is a common deadline. Of course then you'd want x days prior... Hmmm.